packages/sdk/src/dataProtector/fetchGrantedAccess.ts

```diff
@@ -15,6 +16,8 @@ export const fetchGrantedAccess = async ({
   protectedData = 'any',
   authorizedApp = 'any',
   authorizedUser = 'any',
+  isAppStrict = false,
+  isUserStrict = false,
```

Comment:
We might need to mention these 2 new input parameters to the changelog?

Reply:
yes good catch